sstables_loader.cc

```diff
@@ -749,8 +748,16 @@ future<> sstables_loader::download_task_impl::run() {
     };
     llog.debug("Loading sstables from {}({}/{})", _endpoint, _bucket, _prefix);
 
+    // streamer guarantees topology stability, for correctness, by holding effective_replication_map
+    // throughout its lifetime.
+    auto table_id = _loader.local()._db.local().find_uuid(_ks, _cf);
+    std::vector<locator::effective_replication_map_ptr> erms(smp::count);
```

Comment:
These need to be `foreign_ptr<locator::effective_replication_map_ptr>` similar to `global_static_effective_replication_map`, so they would be destroyed on their owner shards on the error-free path.